pokemon.go

```diff
@@ -1,11 +1,110 @@
 package pokemonbattlelib
 
+import "fmt"
+
 type Pokemon struct {
-	// National Pokedex Number
-	NatDex uint16
-	// TODO
+    NatDex uint16                    // National Pokedex Number
+    Nickname string                  // player-given name for the Pokemon
```

Comment:
For simplicity, lets forgo nicknames on pokemon. it's something we can add in later if needed.